src/main/java/edu/cornell/kfs/fp/CuFPConstants.java

```diff
@@ -65,6 +65,12 @@
     public static final String GENERAL_LEDGER_PENDING_ENTRIES_TITLE = "General Ledger Pending Entries";
     public static final String LEFT_PARENTHESIS = "(";
     public static final String RIGHT_PARENTHESIS = ")";
+    
+    public static final String CORPORATE_BILLED_CORPORATE_PAID_DOCUMENT_TYPE_CODE = "CBCP";
+    public static final String CORPORATE_BILLED_CORPORATE_PAID_ERROR_EMAIL_SUBJECT = "Error occurred during Corporate Billed Corporate Paid batch upload process";
+    public static final String CORPORATE_BILLED_CORPORATE_PAID_ERROR_EMAIL_BODY_STARTER = "Errors occured during the Corporate Billed Corporate Paid upload process."; 
+    
+    public static final String DONE_FILE_EXTENSION = ".done";
```

Comment:
Do we need a constant for .done in this FP constants file? I would expect one already exists in KFS.
Is there specific code in this new batch job that deals with .done files that is different from the base KFS code for .done files?

Reply:
I couldn't find a ".done" constant in base code or in our overlay.  I based CBCP job on the pcard document job, and it managed the done files in this manner.